contracts/guard/Guardable.sol

```diff
@@ -24,8 +24,4 @@ contract Guardable is OwnableUpgradeable {
         guard = _guard;
         emit ChangedGuard(guard);
     }
-
-    function getGuard() external view returns (address _guard) {
-        return guard;
-    }
```

Comment:
ok, now this needs a major version bump. but good to remove this obsolete getter 👍

Reply:
Older solidity version were not generating getters for public fields, or why was this getter there?  🤔

Reply:
I realized that we can't remove this, otherwise it makes our guards incompatible with the Safe.